libraries/tests/fixtures.py

```diff
@@ -226,3 +226,19 @@ def github_library():
         "maintainers": ["Tester Testerson <tester -at- example.com>"],
         "cxxstd": "03",
     }
+
+
+@pytest.fixture
+def github_action_boostdep_output_artifact():
+    """
+    A shortened example of the content in the artifact output by the
+    dependency_report.yaml action
+    """
+    return """Dependencies for version boost-1.33.0
+Dependencies for version boost-1.34.0
+Dependencies for version boost-1.35.0
+algorithm -> concept_check config detail logic numeric~conversion
+Dependencies for version boost-1.85.0
+algorithm -> array assert bind concept_check config core
+numeric~conversion -> array
+callable_traits ->"""
```

Comment:
You can use `textwrap.dedent` to make this a bit nicer.

```suggestion
    return dedent(
        """\
        Dependencies for version boost-1.33.0
        Dependencies for version boost-1.34.0
        Dependencies for version boost-1.35.0
        algorithm -> concept_check config detail logic numeric~conversion
        Dependencies for version boost-1.85.0
        algorithm -> array assert bind concept_check config core
        numeric~conversion -> array
        callable_traits ->"""
    )
```

Reply:
Never used that before.  Good to know.